### ISSUE TITLE
Improve color contrast on comparison page

### ISSUE THREAD

**Issue by cassidoo** [npmx-dev/npmx.dev#789]
Right now the color contrast is particularly low on the comparison table page. The yellow stood out to me but both the yellow and green colors fail the WCAG ratio checker:

<img width="922" height="502" alt="Image" src="https://github.com/user-attachments/assets/e94dcad1-78aa-4544-9e54-1bfeaf27cc2b" />

The contrast could be increased with a "badge" styling instead of text only.

Example:

<img width="890" height="440" alt="Image" src="https://github.com/user-attachments/assets/39d8d4bf-6ee5-4ee8-8528-9150c3fc2a7e" />

**Comment by cassidoo** [comment]
Discussed in Discord, this is where we ended up!
<img width="895" height="678" alt="Image" src="https://github.com/user-attachments/assets/931e7613-f0c9-4f31-9d52-b1cbaa45957a" />
